psi4/driver/schema_wrapper.py

```diff
@@ -541,14 +541,8 @@ def run_json_qcschema(json_data, clean, json_serialization, keep_wfn=False):
 
     # Handle special properties case
     if json_data["driver"] == "properties":
-        if "properties" in json_data["model"]:
-            kwargs["properties"] = [x.lower() for x in json_data["model"]["properties"]]
```

Comment:
Do we still need a case change?

Reply:
we shouldn't as far as I'm concerned -- the driver passes the kwargs along and the method runs as per usual, including input sanitization. (plus if there's a case issue, it would probably be better to catch it at the driver level so that it's fixed for any kind of input, not just JSON)